### PR TITLE
Add recipe for Python package `songdkl`

### PR DESCRIPTION
**Pull request by NickleDave**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [n/a] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

I'm the maintainer and I am willing to be listed in the maintainer section :smirk_cat: 
Thank you forgers :pray:

### CODE REVIEW

File: recipes/songdkl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/songdkl-{{ version }}.tar.gz
+  sha256: 3123ba2e27aa898833cdf4a20672962369503acd334791f64adec25acb7dd7bb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (dopplershift) [inline recipes/songdkl/meta.yaml:15]: This should fix Windows:
```suggestion
  number: 0
  entry_points:
    - songdkl = 'songdkl.__main__:main'
```

Comment (NickleDave) [reply recipes/songdkl/meta.yaml:15]: Thank you @dopplershift!

**Comment by NickleDave** [comment]
Hi @conda-forge-admin this is failing on linux with the following:
```
+ songdkl --help
  File "/home/conda/staged-recipes/build_artifacts/songdkl_1670773800364/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol/bin/songdkl", line 11
    sys.exit(main'())

```
from logs here:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=626792&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823

It happens when running the `test` step.

Looks like the error is caused by a typo in `sys.exit(main'())` (apostrophe before parentheses calling main).  
Is that something I did somehow?

File: recipes/songdkl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  entry_points:
+    - songdkl = 'songdkl.__main__:main'
```
Comment (hmaarrfk) [inline recipes/songdkl/meta.yaml:17]:
```suggestion
    - songdkl = songdkl.__main__:main
```

Comment (hmaarrfk) [reply recipes/songdkl/meta.yaml]: I used this recpe as an example: https://github.com/conda-forge/spyder-feedstock/blob/main/recipe/meta.yaml#L17

**Pull request merged**